src/index.ts

```diff
@@ -1,11 +1,49 @@
 import { Probot } from "probot";
 
 export = (app: Probot) => {
+  
+  // Respond to an issue opened 
   app.on("issues.opened", async (context) => {
     const issueComment = context.issue({
       body: "Thanks for opening this issue!",
     });
     await context.octokit.issues.createComment(issueComment);
+  });  
+  // Respond to the GitHub app installation 
+  app.on("installation.created", async (context) =>{
+
+    // TODO: get the installation token from the url
+    const installationTokenUrl = context.payload.installation.access_tokens_url;
+    const installationToken = await context.octokit.request("Get {url}",{url: installationTokenUrl});
+    app.log.info("installationt token: {}",installationToken);
+    // TODO: iterate over the list of repositories
+    const repos = context.payload.repositories;
+    const orgName = repos[0].full_name.split("/")[0];
+
+    const k8s = require('@kubernetes/client-node');
+    const kc = new k8s.KubeConfi();
+    kc.loadFromDefault();
+    const k8sApi = kc.makeApiClient(k8s.coreV1Api);
+    const k8sContext = kc.getCurrentContext();
+    const namespace = k8sContext.split('/')[0];
```

Comment:
Can we move this into a global scope? As global constants? We'll need to access the `k8sApi` and `namespace` in other handlers as well, so we may want to have it declared once only.

Also, `require()` calls should be on top, like `import` statements in Python.

Additionally, can you please add/change the `namespace` to the callback as shown here: https://github.com/open-services-group/peribolos-as-a-service/issues/6#issuecomment-1077951189

It's because the `k8sContext` returns namespace only in case when run locally, when it's running in a container on a cluster, the `kc.getCurrentContext()` returns ` 'inClusterContext'` string. In such cases we need to read a file from mounted service account secret as shown in the comment linked above.